clustered/common/src/main/java/org/ehcache/clustered/common/messages/ServerStoreOpMessage.java

```diff
@@ -107,6 +165,17 @@ public ServerStoreOp operation() {
     public ByteBuffer getPayload() {
       return payload;
     }
+
+    @Override
+    public byte[] encode() {
+      byte[] encodedMsg = super.encode();
```

Comment:
This method looks very inefficient to me: multiple `byte[]` allocations and copies look like this will noticeably impact performance.